lib/suspenders.rb

```diff
@@ -3,5 +3,6 @@
 require 'suspenders/generators/static_generator'
 require 'suspenders/generators/stylesheet_base_generator'
 require 'suspenders/actions'
-require "suspenders/adapters/heroku"
+require 'suspenders/adapters/heroku'
+require 'suspenders/adapters/ci'
```

Comment:
Is it okay to use single-quoted here since that's what the rest of the file uses?